docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,166 @@
+
+After creating your first tracking scenario, a list of tracking scenarios will appear on the page.
+
+![](images/ts-4.png)
+
+Until published, the scenario status is set to "Draft" to represent that it is a work in progress. During this time, you may wish to share it with others in the business to get feedback or approval. To do this, select the link to the tracking scenario and click the "Share" button located at the top right of the page. Please note that others will need access to the Snowplow Console Data Structures section to view the scenario.
+
+* * *
+
+## Adding more details and editing a Tracking Scenario
+
+To add more information or modify a tracking scenario, select it from the appropriate list view in the "Data Structure and Tracking Scenarios" section. This will open a new page displaying the details of the selected tracking scenario.
+
+![](images/ts-5.png)
+
+To make changes, edit the information in each section, and then click the ‘Save and Update’ button to save your changes.
+
+
+### Implementation rules
+
+The Implementation Rules section is where you can specify how each property for this event should be populated.
+
+To add multiple implementation rules, click the **'+ Add implementation rules'** button in the Implementation Rules section. This will open a dialog view.
+
+The dialog displays the list of properties defined in the data structure associated with this scenario.
+
+![](images/ts-7.png)
+
+You can click on any of the properties shown in the list to access a new view inside the dialog box. Here, you will be able to configure the rule's Type of condition, Value, and Comments properties.
+
+![](images/ts-8.png)
+
+Click **'Add rule'** to save the rule, or click the **'Cancel'** button to discard changes.
+
+After configuring the rules, click **'Save and update rules'** to confirm the changes or click **'Close'** to discard them.
+
+You can edit or remove rules in the tracking detail view's rule list.
+
+![](images/ts-9.png)
+
+### Entities
+
+In this section, you can define which [Entities](/docs/understanding-tracking-design/understanding-events-entities/index.md) should be tracked as context for this event.
+
+To add multiple entities, click the **'Add entities'** button. A new dialog box will appear to select entities. This includes both entities from Iglu Central and your own custom entities.
+
+![](images/ts-10.png)
+
+To view detailed information about the properties of an entity, click **'View properties'**. You can use the search bar to find an entity by name.
+
+Once you have selected the entities you want to add to the tracking scenario, click the **'Select entities'** button to confirm, or the **'Cancel'** button to discard your selections. On the next screen, you can configure additional rules. For example, you can specify how many instances of an entity should be tracked with this event, or whether the entity is optional or should always be tracked.
+
+![](images/ts-11.png)
```

Comment:
This screenshot is not up to date - The text on the button is different

Reply:
I still see `Save entities` and `Cancel` buttons for this screenshot on Next

Reply:
I think the first window says "Select Entities" rather than Save, and then the window with the Cardinality says "Save Entities"

Reply:
Updated